internal/util/fileutil.go

```diff
@@ -38,7 +39,7 @@ func RemoveFileFromListOfFileInfo(filename string, files []os.FileInfo) (bool, [
 	return true, files[:len(files)-1]
 }
 
-// Check if a file exists at path
+// FileExists Check if a file exists at path
```

Comment:
are the function names automatically inserted by the linting? If so, we may want to correct the rest of the comments, lower case, third person, ...

Reply:
It is not added by lint, I added this manually, since the IDE complains about the comment not following the go lang pattern. 
I can open a new PR to address these comments :)

Reply:
I don't think it's worth another PR, but I would either leave it out of this one, or fix the grammar to bring it up to style as in the examples here: https://www.digitalocean.com/community/tutorials/how-to-write-comments-in-go#doc-comments